central/config/service/service.go

```diff
@@ -33,10 +33,12 @@ var (
 			// This endpoint should therefore remain public.
 			"/v1.ConfigService/GetPublicConfig",
 		},
+		user.With(permissions.View(resources.VulnerabilityManagementRequests)): {
+			"/v1.ConfigService/GetVulnerabilityExceptionConfig",
+		},
```

Comment:
This way someone with READ access to Administration will continue to have the permission, but someone with `VulnerabilityManagementRequests` will be able to use the API too.
```suggestion
               or.Or(
                   user.With(permissions.View(resources.VulnerabilityManagementRequests)),
                   user.With(permissions.View(resources.Administration))): {
			"/v1.ConfigService/GetVulnerabilityExceptionConfig",
		},
```